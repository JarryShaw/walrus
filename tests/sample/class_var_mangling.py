class A:
    print(__x := 666)
    print(__x)
    print(_A__x)
    print(_A__x := 777)
    print(__x)


print(A.__dict__.get('__x'))
print(A._A__x)
