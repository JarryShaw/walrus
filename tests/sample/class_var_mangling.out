666
666
666
777
777
None
777
